arch/M68K/M68KDisassembler.h

```diff
@@ -0,0 +1,24 @@
+/* Capstone Disassembly Engine */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2014 */
```

Comment:
I change it so it's like this

```C
/* Capstone Disassembly Engine */
/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2014 */
/* M68K Backend by Daniel Collin <daniel@collin.com> */
```

or should I fully remove your name in the m68k files?